### PR TITLE
[#WA-3.9] [#WA-3.12] preparation for sprint 3

### PR DESCRIPTION
**Pull request by repovic48**

### CODE REVIEW

File: WebApplication/Backend/Repositorys/IRegistrationRepository.cs [hunk 1]
```diff
@@ -12,5 +14,6 @@ public interface IRegistrationRepository
         public String GetPatientId(string idd);
         public bool IsPatientIdValid(string patientId);
         public bool ConfirmEmailUpdate(string patientId);
+        public List<FamilyDoctorDTO> GetAllGeneralPractitionerPhysitions();
```
Comment (milicasiriski) [inline WebApplication/Backend/Repositorys/IRegistrationRepository.cs]: Physitions -> Physicians

File: WebApplication/Backend/Repositorys/PhysitianRepository.cs [hunk 2]
```diff
@@ -32,16 +32,16 @@ private List<Physitian> GetPhysitians(String query)
...
-                entity.Contact = (string)sqlReader[5];
-                entity.Email = (string)sqlReader[6];
-                entity.Address = addressRepository.GetAddressBySerialNumber((string)sqlReader[7]);
-                entity.Password = (string)sqlReader[8];
-                entity.Specialization = specializationRepository.GetSpecializationsBySerialNumber((string)sqlReader[9]);
-                entity.VacationTime = timeIntervalRepository.GetTimeIntervalsById((string)sqlReader[10]);
-                entity.WorkSchedule = timeIntervalRepository.GetTimeIntervalById((string)sqlReader[11]);
-                entity.AllSpecializations = specializationRepository.GetSpecializationsNameBySerialNumber((string)sqlReader[12]);
-                entity.AddressSerialNumber = (string)sqlReader[13];
+                //entity.DateOfBirth = (DateTime)sqlReader[4];
```
Comment (milicasiriski) [inline WebApplication/Backend/Repositorys/PhysitianRepository.cs]: Delete if it's unused

File: WebApplication/Backend/Services/RegistrationService.cs [hunk 3]
```diff
@@ -45,5 +48,10 @@ public bool ConfirmEmailUpdate(string id)
             return registrationRepository.ConfirmEmailUpdate(id);
         }
 
+        public List<FamilyDoctorDTO> GetAllPhysitians()
+        {
+            return registrationRepository.GetAllGeneralPractitionerPhysitions();
```
Comment (milicasiriski) [inline WebApplication/Backend/Services/RegistrationService.cs]: Physicians

File: WebApplication/Migrations/20201203134149_migration1.Designer.cs [hunk 4]
```diff
@@ -0,0 +1,1422 @@
+﻿// <auto-generated />
+using System;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Migrations;
+using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
+using WebApplication.Backend.Model;
+
+namespace WebApplication.Migrations
```
Comment (milicasiriski) [inline WebApplication/Migrations/20201203134149_migration1.Designer.cs]: Delete

File: WebApplication/wwwroot/app/registration.js [hunk 5]
```diff
@@ -31,10 +32,22 @@
 				image: null,
 				guest: false,
 				emailConfirmed: false,
-				sucessFlag: false,
+				chosenDoctor:null,
+				//sucessFlag: false,
```
Comment (milicasiriski) [inline WebApplication/wwwroot/app/registration.js]: delete if unnecessary

**Review by milicasiriski — CHANGES_REQUESTED** [review]
Good PR! Please delete .idea folder and migrations

**Review by milicasiriski — APPROVED** [review]

**Review by lukaDoric — APPROVED** [review]

**Review by VladimirBudjen — APPROVED** [review]

**Pull request merged**